frontend/pages/discover/open-calls.tsx

```diff
@@ -83,7 +83,7 @@ const OpenCallsPage: PageComponent<OpenCallsPageProps, DiscoverPageLayoutProps>
                 <Loading visible={loading} iconClassName="w-10 h-10" />
               </span>
             )}
-            <div className="grid grid-cols-1 gap-6 p-1 md:grid-cols-2">
+            <div className="grid grid-cols-1 gap-2 p-1 sm:gap-6 md:grid-cols-2">
```

Comment:
For consistency with all the other card types, let's set the same padding:
```suggestion
            <div className="grid grid-cols-1 gap-2 p-0.5 sm:gap-6 md:grid-cols-2">
```